docs/rules/meta-property-ordering.md

```diff
@@ -0,0 +1,51 @@
+# enforce ordering of meta properties in rule source (meta-property-ordering)
+
+(fixable) The `--fix` option on the [command line](../user-guide/command-line-interface#fix) automatically fixes problems reported by this rule.
+
+This rule enforces that the properties of rule meta are arranged in a consistent order.
+
+## Rule Details
+
+### Options
+
+This rule has an array option:
+
+* `['type', 'docs', 'fixable', 'schema', 'messages', 'deprecated', 'replacedBy']` (default): The properties of meta should be placed in a consistent order.
+
+Examples of **incorrect** code for this rule:
+
+```js
+
+/* eslint eslint-plugin/meta-property-ordering: ["error",
+  ["type", "docs", "fixable", "schema", "messages"]
+] */
+
+// invalid; wrong order
+{
```

Comment:
Can you update these examples to include a full rule with the `meta` property, rather than just being an object?

Reply:
updated as suggested.